modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx

```diff
@@ -64,7 +64,11 @@ export default function LeftSidebar({setShowModal}: LeftSidebarProps) {
 			(key) => key === selectedObjectFolder.name
 		) as string;
 
-		setExpandedKeys(new Set([selectedObjectFolderKey]));
+		const filteredFolders = leftSidebarItems
+			.filter((item) => item.leftSidebarObjectDefinitionItems?.length)
+			.map((filteredNames) => filteredNames.name);
```

Comment:
`filteredNames` is not an array of names, but and array of items, returned by the filter array method. I would name this `filteredItem`, in the singular.